### ISSUE TITLE
[BUG] - play({local: true}) is not local

### ISSUE THREAD

**Issue by MrVauxs** [fantasycalendar/FoundryVTT-Sequencer#281]
**Describe the bug**
Animations played with play({local: true}) are not played locally, but globally (other users see it).

**To Reproduce**
```js
new Sequence()
        .effect()
            .file("jb2a.shield.03.complete")
            .attachTo(token)
            .scaleToObject(1.5)
            .persist()
        .play({local: true})
```
Play with 2 windows open on different accounts. See that the animation is not local.

**Expected behavior**
The Sequence being local and only visible to one user.

**Screenshots**

https://github.com/user-attachments/assets/de47c008-b1ee-4a9c-84ba-64cefc6107be

**Setup:**
 - Foundry Version: 12.330
 - System Version: pf2e 6.1.2
 - Sequencer Version: 3.2.11

**Active modules:**
 - Sequencer
 - socketlib

**Comment by MrVauxs** [comment]
This happened because of [git](https://github.com/fantasycalendar/FoundryVTT-Sequencer/commit/25e249ab2fa556469a1400fd02c3366af0e3e8f2).

![chrome_pcRO3eYsZq](https://github.com/user-attachments/assets/eb1967be-ab4f-46ad-b7cc-947292e50f02)

**Comment by Haxxer** [comment]
Fixed in 3.2.12
